pages/upgrade/rolling_es_upgrade.rst

```diff
@@ -0,0 +1,32 @@
+.. _es_rolling_upgrade:
+
+***********************************
+Elasticsearch Rolling Upgrade Notes
+***********************************
+
+This page contains a few notes and a recommended procedure to perform a rolling upgrade for a Elasticsearch cluster utilized by Graylog.
+
+Elasticsearch supports rolling upgrades to avoid downtimes during upgrades. Detailed information about the procedures and limitations are provided `here <https://www.elastic.co/guide/en/elasticsearch/reference/master/rolling-upgrades.html>`__.
+
+Graylog supports rolling upgrades without restarting any Graylog node for Elasticsearch upgrades *between minor versions*. While Elasticsearch supports upgrading from e.g. 6.8 to 7.0, Graylog requires a restart when the major version of the Elasticsearch cluster is changed.
+
+To avoid message loss in case of a rolling Elasticsearch upgrade which requires the restart of Graylog nodes, please follow this procedure:
```

Comment:
Would this scenario be a little easier if we told people to stop message processing?
I think in that case we could wholesale update the ES cluster in one go, and then restart the Graylog nodes one-by-one to ensure message delivery.

Afterwards, enabling processing again should not lose any messages as well, if at the cost of having a brief interruption in ability to search. Maybe mention it as a second option?

Reply:
Yes, that would work as well. Shutting down the Graylog node just does not need an explanation. But maybe we have a good part in the docs I could link to for stopping message processing?

Reply:
hm, appears we don't actually. Let's keep it the way it is, probably better to have a single option than multiple ones that confuse the matter further.